JenkinsFile_build_doc.groovy

```diff
@@ -84,21 +96,46 @@ timeout(time: 6, unit: 'HOURS') {
                             sh "tar -zxf ${ARCHIVE}"
                         }
 
-                        dir('eclipse_repo') {
-                            git branch: BRANCH, url: ECLIPSE_REPO
-                            sh 'rm -rf *'
-                            sh "cp -r ${WORKSPACE}/built_doc/site/* ."
-                            sh 'git status'
-                            STATUS = sh (script: 'git status --porcelain', returnStdout: true).trim()
-                            if ("x${STATUS}" != "x") {
-                                sh 'git add -A'
-                                sh "git commit -m 'Generated from commit: ${MERGE_COMMIT}'"
-                                sh "git push origin ${BRANCH}"
-                            }
-                        }
+						if (params.ghprbPullId) {
+							dir('openj9-staging') {
+								git branch: BRANCH, url: OPENJ9_STAGING
+								// SUE: Was `sh 'rm -rf *'`
+								// SUE: Check if the PR is already staged. If so, remove content. Else create a new directory. 
+								sh """
+									if [ -d "${ghprbPullId}" ]
+									then
+										rm -R openj9-docs-staging/${ghprbPullId}/*   
+									else
+										mkdir openj9-docs-staging/${ghprbPullId}
+									fi
+								"""	
```

Comment:
This check can be simplified using pipeline steps
```
dir("${ghprbPullId}") {
    deleteDir()
    sh "cp -r ${WORKSPACE}/built_doc/site/* ."
}
sh 'git status'
```
[Pipeline: Basic Steps](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Basic+Steps+Plugin)
>Recursively deletes the current directory and its contents. Symbolic links and junctions will not be followed but will be removed. To delete a specific directory of a workspace wrap the deleteDir step in a dir step.

Reply:
Will that step create the PR directory if it does not exist? The docs weren't clear.

Reply:
Yes it will.